src/ray/core_worker/test/memory_store_test.cc

```diff
@@ -44,7 +44,7 @@ TEST(TestMemoryStore, TestReportUnhandledErrors) {
   // Check delete after get.
   RAY_CHECK(provider->Put(obj1, id1));
   RAY_CHECK(provider->Put(obj1, id2));
-  provider->Get({id1}, 1, 100, context, false, &results);
+  RAY_UNUSED(provider->Get({id1}, 1, 100, context, false, &results));
```

Comment:
Is this intentional?

Reply:
Yes. I found this test was failing on Mac always for some reason.